src/libtrx/game/objects/names.c

```diff
@@ -21,6 +22,9 @@ static void M_ClearNames(void)
     for (GAME_OBJECT_ID object_id = 0; object_id < O_NUMBER_OF; object_id++) {
         M_NAME_ENTRY *const entry = &m_NamesTable[object_id];
         Memory_FreePointer(&entry->name);
+        if (entry->description != NULL) {
+            Memory_FreePointer(&entry->description);
+        }
```

Comment:
No need for the NULL check.

Reply:
Thank you, slipped my mind while reviewing 😄 Did this part a few days ago.